test/Interpreter/asan-globals-overwrite.swift

```diff
@@ -0,0 +1,19 @@
+// RUN: %target-build-swift -O -whole-module-optimization -sanitize=address %s -o %t
+// RUN: %target-codesign %t
+// RUN: %target-run %t
```

Comment:
I think this is `%target-run-simple-swift(-O -whole-module-optimization -sanitize=address)`.